stages/org.osbuild.yum.config

```diff
@@ -26,13 +28,73 @@ SCHEMA = r"""
         "description": "Determines how upstream HTTP caches are instructed to handle any HTTP downloads that YUM does."
       }
     }
+  },
+  "plugins": {
+    "additionalProperties": false,
+    "type": "object",
+    "description": "YUM plugins configuration",
+    "properties": {
+      "langpacks": {
+        "additionalProperties": false,
+        "type": "object",
+        "description": "'langpacks' YUM plugin configuration",
+        "properties": {
+          "locales": {
+            "type": "array",
+            "description": "list of locales for YUM",
+            "items": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    }
   }
 }
 """
 
 
+def configure_plugins(tree, plugins_options):
+    for plugin, plugin_options in plugins_options.items():
+        # don't touch the configuration file if there is nothing to do
+        if not plugin_options:
+            continue
+
+        plugin_conf_path = f"{tree}/etc/yum/pluginconf.d/{plugin}.conf"
+        plugin_conf = iniparse.SafeConfigParser()
+
+        try:
+            with open(plugin_conf_path, "r") as f:
+                plugin_conf.readfp(f)
+        except FileNotFoundError:
+            print(f"Error: {plugin} configuration file '{plugin_conf_path}' does not exist, will create it.")
```

Comment:
Nitpick: I used `Warning: ...` in the log message on line 85. I would be probably nice to be consistent and use `Error:` or `Warning:` in both places... But also feel free to keep it as is.